docs/lib/graphqlapi/fragments/android/getting-started.md

```diff
@@ -112,6 +112,9 @@ Also in your **app** `build.gradle`, add this piece of code to support the Java
 ```groovy
 android {
   compileOptions {
+    // Enable support for the new language APIs
+    coreLibraryDesugaringEnabled true
+    // Enable Java 8 features
     sourceCompatibility JavaVersion.VERSION_1_8
     targetCompatibility JavaVersion.VERSION_1_8
   }
```

Comment:
This block also needs to be added, otherwise the build will fail (after end of android block):
```
project.dependencies {
    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.0.10'
}

```

Reply:
+1, if we do need de-sugaring in the downstream project, we should also document this required dependency.